### DISCUSSION TITLE
2024 Summit in Czechia: July 1-5  *save the date*

### DISCUSSION THREAD

**Discussion by fcbond** [delph-in/docs#145] [Announcements]
Dear all,

We are pleased to announce that the 2024 Summit will be hosted by Palacký Unviersity in Olomouc, Czechia, from July 1-5.   It will be followed by a [Workshop on Formal Approaches to Under-resourced Languages](https://bond-lab.github.io/2024-hpsg/workshop.html) (July 8) and the [31st Conference on HPSG](https://bond-lab.github.io/2024-hpsg/) (July 9-10).

You can find information about the event at this wiki page:

https://github.com/delph-in/docs/wiki/OlomoucTop

We are looking forward to seeing many of you there in person + many virtually as well. 

The participants page is ready, please sign up!

We will be sending further messages asking for program input closer to the event. 

Finally, please consider who else at your site might be interested in participating and let them know too.

Francis,
for the Standing Committee

**Comment by fcbond** [comment]
Hi, I'm not sure if this went to everyone so tagging all @delph-in/participants 

Can you respond (emoji is fine), if you see this thread?

**Comment by fcbond** [comment]
Dear all,

Sorry for sending this multiple times, trying to get all the information to everyone seamlessly!

We are pleased to announce that the 2024 Summit will be hosted by Palacký Unviersity in Olomouc, Czechia, from July 1-5. It will be followed by a Workshop on Formal Approaches to Under-resourced Languages (July 8) and the 31st Conference on HPSG (July 9-10).

You can find information about the event at this wiki page:

https://github.com/delph-in/docs/wiki/OlomoucTop

We are looking forward to seeing many of you there in person + many virtually as well.

The participants page is ready, please sign up!

We will be sending further messages asking for program input closer to the event.

Finally, please consider who else at your site might be interested in participating and let them know too.

Francis,
for the Standing Committee

P.S. To all  @delph-in/participants

**Comment by petterha** [comment]
👍

tor. 29. feb. 2024, 12:31 skrev Francis Bond ***@***.***>:

**Comment by fcbond** [comment]
Hi, 

Just a quick reminder to make your plans for [the Olomouc Summit](https://github.com/delph-in/docs/wiki/OlomoucTop) (and maybe [HPSG 2024](https://bond-lab.github.io/2024-hpsg/)).  

If you can come, please add your name to the [participants list](https://delph-in.github.io/docs/summits/OlomoucParticipants/) so we can get an idea of numbers.

Hoping to see you in July,

Francis

P.S. Tagging all @delph-in/participants

**Comment by danflick** [comment]
Greetings @delph-in/participants

We the newly formed program committee for the 2024 Summit invite you to send us your proposed contributions for the program in Olomouc in the first week of July.

Please email your suggestions and requests to [delphinsummit@gmail.com](mailto:delphinsummit@gmail.com) by **Friday, 24 May.**  Some possibilities are:

    site updates [5–10 minutes, including questions]
    plenary discussions, with introductory problem statement by moderator [typically 10+50 minutes]
    short, problem-focused presentations (possibly backed up by shared paper pre-print) [10+20]
    long presentations (possibly including an extended Q&A) [20+20–30]
    SIG (Special Interest Group) topics for the last two days of the summit
    offers or requests for tutorials

Also, if you are planning to attend and have not yet added your name to the list of participants, please do so by following the link at
https://github.com/delph-in/docs/wiki/OlomoucTop

Thanks, and we look forward to hearing from you soon!

Dan Flickinger and Guy Emerson

**Comment by danflick** [comment]
Hi again, @delph-in/participants

We the program committee for the DELPH-IN 2024 Summit have now posted the draft program at
https://github.com/delph-in/docs/wiki/OlomoucSchedule
If you will be attending, and sent us a content proposal at [delphinsummit@gmail.com](mailto:delphinsummit@gmail.com), please check the program page to confirm that we have reflected your intentions accurately.

We look forward to seeing everyone in Olomouc!

Dan and Guy

**Comment by danflick** [comment]
Got it.

**Comment by fcbond** [comment]
G'day @delph-in/participants,

Some updates from the local organizers:

 * We will be making a T-shirt!  Please fill in your size on the [participants page](https://github.com/delph-in/docs/wiki/OlomoucParticipants) as soon as possible (or you will end up with an M).   Dimensions [here](https://onlinecatalog.malfini.com/file/pdf/size_chart/129---product_size.pdf)
   * If you want additional T-shirts please note.  They will be around ~200~ 300 CZK.

 * There will be a registration fee this year, payable in cash when you arrive.  We would prefer Czech Crowns but will also accept Euros.  The exchange rate is rough, but we needed an even number to make change easily.  Registration covers coffee/juice/snacks during the day, stand up/karaoke on Monday (but not drinks), Excursion on Wednesday, Banquet on Thursday, and one T-Shirt.   
     *  Virtual participation is free.
     * Can you indicate if you are a student on the [participants page](https://github.com/delph-in/docs/wiki/OlomoucParticipants)  in the comments (to help us budget).

| Type  | Czech Crowns | Euros | 
|-------|------: | -----: | 
| General | 3,500 | 150 |
| Student | 2,500 | 100 | 

 * Please add to the participants page if you are planning to come to the banquet and excursion (and if so how many people).  
  If with kids, please add X+Y (for adults and kids).
* The excursion will be to   [Helfštýn Castle](https://helfstyn.cz/en).  We will have a guided tour and then a couple of free hours to wander around (there are cafes and ice-cream)
* In previous summits, people requested name-tags.  Please bring a name tag if you already have one (as most of us do), we will have some spare ones on site.

* We have no external funding for the summit, so we will be using the department's plates, cups and so forth.  Please pitch in with clean up at the end of each day!

Very much looking to seeing you all,

Francis and Joanna

**Comment by fcbond** [comment]
Ahoj @delph-in/participants,

I hope that this will be the last message before we see you.  We are getting excited here.

- If you are coming but have not added your name to the [participants page](https://github.com/delph-in/docs/wiki/OlomoucParticipants) please do so at once!
- If you have any dietary requirements/preferences, please add them as comments to the participants page or email me.
- Next week may be warm at first (high of 30°C / 86°F) and then drop a bit  (low of 12°C / 54°F) with probable rain, so bring a range of clothing 
- Please bring cash for the registration
- Please bring a name tag
- Please bring a hunger for knowledge!

Vítejte v Olomouci,

Francis and Joanna

**Comment by guyemerson** [comment]
@delph-in/participants Anyone else free for dinner tonight?

**Comment by fcbond** [comment]
Hi,

I am meeting Michael Goodman and family at Long Story Short at 7pm.

It is a nice restaurant, with plenty of room.
https://www.longstoryshort.cz/

Any one who would like to join is welcome!


On Sun, 30 Jun 2024 at 12:57, Guy Emerson ***@***.***> wrote:

> Anyone else free for dinner tonight?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/145#discussioncomment-9916278>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRRJNRWEGPY4RTRX3ALZJ7QCNAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJWGI3TQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Reply by guyemerson** [reply]
Great, I'll see you there!

**Comment by emilymbender** [comment]
I'll see if I can stay awake that long. (Waiting for my train in Praha now.)

**Comment by arademaker** [comment]
In the bus @emily going to the train station in Prague too.

**Comment by anncopestake** [comment]
I think I'll be a bit late for a 7pm dinner but I expect could do 8pm if there's another group.

**Comment by fcbond** [comment]
I think people can just drift in and out, but I am going to try not to stay
up too late, ...

On Sun, 30 Jun 2024 at 14:37, anncopestake ***@***.***> wrote:

> I think I'll be a bit late for a 7pm dinner but I expect could do 8pm if
> there's another group.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/145#discussioncomment-9916675>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRVEMBIDI2DOOD6GUFDZJ73XHAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJWGY3TK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by EricZinda** [comment]
I’ll try to join at 7. On the bus with Tara now

eric

On Jun 30, 2024, at 3:00 PM, Francis Bond ***@***.***> wrote:

﻿

I think people can just drift in and out, but I am going to try not to stay
up too late, ...

On Sun, 30 Jun 2024 at 14:37, anncopestake ***@***.***> wrote:

> I think I'll be a bit late for a 7pm dinner but I expect could do 8pm if
> there's another group.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/145#discussioncomment-9916675>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRVEMBIDI2DOOD6GUFDZJ73XHAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJWGY3TK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


--
Francis Bond <https://fcbond.github.io/>

**Comment by petterha** [comment]
I'll be there at 7. (I'm already checked in at Hotel Palac.)

Petter

søn. 30. jun. 2024, 15:16 skrev EricZinda ***@***.***>:

> I’ll try to join at 7. On the bus with Tara now
>
> eric
>

**Comment by emilymbender** [comment]
This train is 50 min delayed ... and counting.  I think I'm just going to
crash when I finally make it to my hotel.

See you all tomorrow!

Emily

**Comment by arademaker** [comment]
We are in the same train Emily, not sure what happened.

**Comment by emilymbender** [comment]
Something about "extraordinary lockout" and moving some train, per the
translated info page.

https://www.cd.cz/vlak/893/30.6.2024/5457276/14.52/5434755/19.22/

**Comment by anncopestake** [comment]
Looks like my train has caught up your hold up ...

**Comment by curtosis** [comment]
So jealous to see everyone arriving -- I had to cancel my trip late due to some family  and [new] job needs. Hoping to make some sessions virtually, and for next year!

**Comment by emilymbender** [comment]
Sorry to hear you're stuck too Ann and that you aren't coming afterall
Chris! But at least you can feel smug about not being stuck in an
interminable train delay....

79 minutes and counting and the reaction of the people near me to the
announcement that just aired wasn't reassuring.

**Comment by fcbond** [comment]
Hi,
Luis, I and  are already at the restaurant if anyone is ready already.

On Sun, 30 Jun 2024 at 17:39, emilymbender ***@***.***> wrote:

> Sorry to hear you're stuck too Ann and that you aren't coming afterall
> Chris! But at least you can feel smug about not being stuck in an
> interminable train delay....
>
> 79 minutes and counting and the reaction of the people near me to the
> announcement that just aired wasn't reassuring.
>
> On Sun, Jun 30, 2024, 5:37 PM curtosis ***@***.***> wrote:
>
> > So jealous to see everyone arriving -- I had to cancel my trip late due
> to
> > some family and [new] job needs. Hoping to make some sessions virtually,
> > and for next year!
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/145*discussioncomment-9917564__;Iw!!K-Hz7m0Vt54!izyql125_P13eQmqttfc4u7Be11n-g1j5LMVF8JsFLgw6UWJVqKzAt9hL1R4OgrfpUr8ugbCGbeE4w_A9r5hoXc$>,
>
> > or unsubscribe
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AC5VF6WL2D467BR5SHDWZGDZKAQZ7AVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJXGU3DI__;!!K-Hz7m0Vt54!izyql125_P13eQmqttfc4u7Be11n-g1j5LMVF8JsFLgw6UWJVqKzAt9hL1R4OgrfpUr8ugbCGbeE4w_A21JwE4A$>
>
> > .
> > You are receiving this because you are on a team that was
> mentioned.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/145#discussioncomment-9917594>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRTCUQF7L2KNQL437FTZKARDBAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJXGU4TI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by fcbond** [comment]
Did you make it safely?

On Sun, 30 Jun 2024 at 17:39, emilymbender ***@***.***> wrote:

> Sorry to hear you're stuck too Ann and that you aren't coming afterall
> Chris! But at least you can feel smug about not being stuck in an
> interminable train delay....
>
> 79 minutes and counting and the reaction of the people near me to the
> announcement that just aired wasn't reassuring.
>
> On Sun, Jun 30, 2024, 5:37 PM curtosis ***@***.***> wrote:
>
> > So jealous to see everyone arriving -- I had to cancel my trip late due
> to
> > some family and [new] job needs. Hoping to make some sessions virtually,
> > and for next year!
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/145*discussioncomment-9917564__;Iw!!K-Hz7m0Vt54!izyql125_P13eQmqttfc4u7Be11n-g1j5LMVF8JsFLgw6UWJVqKzAt9hL1R4OgrfpUr8ugbCGbeE4w_A9r5hoXc$>,
>
> > or unsubscribe
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AC5VF6WL2D467BR5SHDWZGDZKAQZ7AVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJXGU3DI__;!!K-Hz7m0Vt54!izyql125_P13eQmqttfc4u7Be11n-g1j5LMVF8JsFLgw6UWJVqKzAt9hL1R4OgrfpUr8ugbCGbeE4w_A21JwE4A$>
>
> > .
> > You are receiving this because you are on a team that was
> mentioned.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/145#discussioncomment-9917594>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRTCUQF7L2KNQL437FTZKARDBAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJXGU4TI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by arademaker** [comment]
Yes, we made it. After 50-60 minutes of waiting, we finally arrived at Olomouc. But only 19:30 hs, Emily and I are both exhausted.

Best,
Alexandre

**Comment by fcbond** [comment]
Glad to hear you made it, have a good rest, and see you tomorrow!

On Sun, 30 Jun 2024 at 21:36, Alexandre Rademaker ***@***.***>
wrote:

>
> Yes, we made it. After 50-60 minutes of waiting, we finally arrived at
> Olomouc. But only 19:30 hs, Emily and I are both exhausted.
>
> Best,
> Alexandre
>
> > On 30 Jun 2024, at 21:23, Francis Bond ***@***.***> wrote:
> >
> >
> > Did you make it safely?
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/145#discussioncomment-9918785>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRVERBBJAMQFL4KUGADZKBM3FAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJYG44DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by danflick** [comment]
Me too, an hour after Emily and Alexandre. See you all in the morning!

Dan

**Comment by lascarid** [comment]
Hi everyone,

I wish I could be there.  Have a gerat meeting.  I don't think I will even be able to participate online (and didn't receive a link anyway).  My mum needs lots of care at the moment, as she heads for eye surgery.  Hopefully I can make it next year!

Cheers,
Alex.
________________________________

**Comment by fcbond** [comment]
Hi Alex,

I hope that your Mum's surgery goes as well as it possibly can.

The zoom link is on the schedule page, I reproduce it here in case anyone
needs it:

https://cesnet.zoom.us/j/97067901742

The schedule is here:

https://github.com/delph-in/docs/wiki/OlomoucSchedule


On Sun, 30 Jun 2024 at 21:47, Alex Lascarides ***@***.***>
wrote:

> Hi everyone,
>
> I wish I could be there. Have a gerat meeting. I don't think I will even
> be able to participate online (and didn't receive a link anyway). My mum
> needs lots of care at the moment, as she heads for eye surgery. Hopefully I
> can make it next year!
>
> Cheers,
> Alex.
> ________________________________
> From: Francis Bond ***@***.***>
> Sent: 30 June 2024 20:37
> To: delph-in/docs ***@***.***>
> Cc: Alex Lascarides ***@***.***>; Team mention ***@***.***>
> Subject: Re: [delph-in/docs] 2024 Summit in Czechia: July 1-5 *save the
> date* (Discussion #145)
>
> This email was sent to you by someone outside the University.
> You should only click on links or attachments if you are certain that the
> email is genuine and the content is safe.
>
> Glad to hear you made it, have a good rest, and see you tomorrow!
>
> On Sun, 30 Jun 2024 at 21:36, Alexandre Rademaker ***@***.***>
> wrote:
>
> >
> > Yes, we made it. After 50-60 minutes of waiting, we finally arrived at
> > Olomouc. But only 19:30 hs, Emily and I are both exhausted.
> >
> > Best,
> > Alexandre
> >
> > > On 30 Jun 2024, at 21:23, Francis Bond ***@***.***> wrote:
> > >
> > >
> > > Did you make it safely?
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/orgs/delph-in/discussions/145#discussioncomment-9918785>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAIPZRVERBBJAMQFL4KUGADZKBM3FAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJYG44DK>
>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
>
> --
> Francis Bond <https://fcbond.github.io/>
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/orgs/delph-in/discussions/145#discussioncomment-9918790>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AJUGPVYZEAVE7KUC6UM3WHLZKBNA3AVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJYG44TA>.
>
> You are receiving this because you are on a team that was
> mentioned.Message ID: ***@***.***>
>
> The University of Edinburgh is a charitable body, registered in Scotland,
> with registration number SC005336. Is e buidheann carthannais a th’ ann an
> Oilthigh Dh?n ?ideann, cl?raichte an Alba, ?ireamh cl?raidh SC005336.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/145#discussioncomment-9918862>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRQUQLEBTQS3ZNCAIS3ZKBOFPAVCNFSM6AAAAABD6RA3ECVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJYHA3DE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by guyemerson** [comment]
@delph-in/participants Give your SIG preferences here: https://docs.google.com/spreadsheets/d/1zPBiB9JAgOdxdZqiWcmVoPU3-uPpllfFQJvImppmFKg/edit?usp=sharing
